### DISCUSSION TITLE
Two documentation versions?

### DISCUSSION THREAD

**Discussion by iainbeeston** [avo-hq/avo#1232] [Feature feedback]
I keep finding two different versions of the avo 2 documentation site. They seem to exist on the same URL and I haven't been able to figure out why sometimes I see one and sometimes I see another, but I find it confusing, especially because they don't have the same information!

Here's one version (the older one) at https://docs.avohq.io/2.0/

![Screenshot 2022-09-14 at 11 09 58](https://user-images.githubusercontent.com/111963/190126858-9e5192cf-3d55-483e-b318-9cccc2882607.jpg)

And here's the other (the newer one) also at https://docs.avohq.io/2.0/

![Screenshot 2022-09-14 at 11 10 00](https://user-images.githubusercontent.com/111963/190127002-6070bfac-0bde-4047-9447-08f8a03cf0c7.jpg)

The only way I've been able to reliably get the newer documentation is to go to avohq.io and click on "Docs" at the top - otherwise it can be fairly random which one I see. I've noticed that the avo newsletter email (the one that announces new features) usually takes me to the old documentation site, which is especially confusing because it doesn't document any of the new features that are coming out!

Is there any chance the old documentation site could be switched off? Or moved to a different url?

**Comment by adrianthedev** [comment]
I've hit this too and the way I resolved it was a hard refresh. My guess is that there's some caching going on there. Possibly on the service worker level. 

I'll look into how to bust that with Vitepress.
